changelog/806.deprecate.rst

```diff
@@ -0,0 +1 @@
+|commands| Deprecate the ``sync_commands``, ``sync_commands_debug``, and  ``sync_commands_on_cog_unload`` parameters of :class:`~disnake.ext.commands.Bot` and :class:`~disnake.ext.commands.InteractionBot`. These have been replaced with the ``command_sync`` parameter which takes a :class:`~disnake.ext.commands.ApplicationCommandSyncFlags` instance.
```

Comment:
```suggestion
|commands| Deprecate the ``sync_commands``, ``sync_commands_debug``, and  ``sync_commands_on_cog_unload`` parameters of :class:`~disnake.ext.commands.InteractionBotBase` and relevant subclasses. These have been replaced with the ``command_sync`` parameter which takes a :class:`~disnake.ext.commands.ApplicationCommandSyncFlags` instance.
```

Reply:
InteractionBotBase is not a documented class, its an implementation detail.